examples/parsing.py

```diff
@@ -26,23 +27,33 @@
     checkpoint, trust_remote_code=True, revision=revision
 ).to(device)
 
-input_text = "def "
-inputs = tokenizer.encode(input_text, return_tensors="pt").to(device)
+sql_grammar_url = "https://github.com/zbrookle/sql_to_ibis/raw/0e9226da42065940ce21439d490f9fcacadc7f92/sql_to_ibis/grammar/sql.lark"
```

Comment:
What do you think about maintaining some of the grammars we need?

Reply:
Yeah, we can add them with the appropriate licenses, of course.